szyfrowanie-kwantowe.md

```diff
@@ -64,15 +189,77 @@ Roznice miedzy bitem a kubitem moznaby wytlumaczyc ponizszym porownaniem:
 
 ---
 
-# Podsumowanie:
+# Przykład użycia (QKD)
 
-Szyfrowanie kwantowe oferuje obiecującą ścieżkę do bezpieczeństwa informacji w erze komputacji kwantowej. Chociaż technologia ta jest wciąż w fazie rozwoju, jej zdolność do zapewnienia bezpieczeństwa komunikacji jest niezrównana w porównaniu z klasyczną kryptografią.
+Najbardziej znanym przykładem jest protokół **BB84**, opracowany przez Charlesa Bennetta i Gillesa Brassarda w 1984 roku.
+
+1. **Wybór Bazy i Stanów Kwantowych**:
+
+   - Nadawca (zwykle nazywany Alicją) generuje losowy ciąg bitów (np. $01011$).
+   - Dla każdego bitu Alicja losowo wybiera jedną z dwóch baz kwantowych:
+     - bazę prostokątną $(1)$
+     - bazę diagonalną $(2)$
+
+---
+
+# Przykład użycia (QKD) c.d.
+
+$$
+\begin{align}
+  0 \text{ jako } |0\rangle \\
```

Comment:
Działa to w marpie? Bo próbowałem to wcześniej zrobić i się nie formatowało mi ładnie :/

Reply:
Tak dziala ladnie, mozliwe ze wtyczka w Pycharmie jest nie halo